src/pages/TodoList.js

```diff
@@ -0,0 +1,210 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import ListContainer from '../components/ListContainer';
+import { TODOKEY } from '../constants/TODOKEY';
+
+const TodoList = () => {
+  const [inputText, setInputText] = useState('');
+  const [todoList, setTodoList] = useState(
+    JSON.parse(localStorage.getItem(TODOKEY)) ?? []
+  );
+  const [numOfTodo, setNumOfTodo] = useState(0);
+  const [numOfDone, setNumOfDone] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState({});
+  const [isVisibleModal, setIsVisibleModal] = useState(false);
+
+  useEffect(() => {
+    setNumOfTodo(todoList.filter((todo) => !todo.isDone).length);
+    setNumOfDone(todoList.filter((todo) => todo.isDone).length);
+  }, [todoList]);
+
+  const handleInputChange = (e) => {
+    setInputText(e.target.value);
+  };
+
+  const handleInputKeyDown = (e) => {
+    if (e.key === 'Enter') {
+      submitInput();
+    }
+  };
+
+  const submitInput = () => {
+    if (inputText.trim()) {
+      const todoObj = {
+        text: inputText, // string
+        time: String(Date.now()), // string,
+        isDone: false, // boolean
+      };
+      setTodoList([...todoList, todoObj]);
+      setInputText('');
+
+      localStorage.setItem(TODOKEY, JSON.stringify([...todoList, todoObj]));
+    }
+  };
+
+  const removeTodo = () => {
+    const tempList = todoList.filter((todo) => todo.time !== selectedTodo.id);
+    setTodoList(tempList);
+
+    localStorage.setItem(TODOKEY, JSON.stringify(tempList));
+
+    setIsVisibleModal(false);
+  };
+
+  return (
+    <Wrapper>
+      <Main>
+        <Title>
+          <span>Things To Do</span>
+        </Title>
+
+        <SectionInput>
+          <input
+            type="text"
+            placeholder="type here ..."
+            value={inputText}
+            onChange={handleInputChange}
+            onKeyDown={handleInputKeyDown}
+          ></input>
+          <span onClick={submitInput}>+</span>
+        </SectionInput>
```

Comment:
이 부분은` <SectionInput>`을 `styled.form`으로 선언하고, `<span>` 부분을 `<button type = 'submit'>`(따로 type을 지정하지 않으면 default값이 submit으로 됩니다) 으로 지정하면 onSubmit함수 하나로 handling할 수 있을 것 같아요!

```suggestion
       <SectionInput> //styled.form으로 정의
          <input
            type="text"
            placeholder="type here ..."
            value={inputText}
            onChange={handleInputChange}
            onSubmit={handleSubmit}
          />
          <button type="submit">+</button>
       </SectionInput>
```

Reply:
이 방법이 더 좋네요! 이렇게 수정해보겠습니다
용도에 맞는 태그를 적절히 활용하는 것으로 코드를 줄일 수 있다는 것 👍👍 앞으로 작업하면서 이 부분도 신경써야겠어요